vendor/github.com/packer-community/winrmcp/winrmcp/winrmcp.go

```diff
@@ -42,7 +43,17 @@ func New(addr string, config *Config) (*Winrmcp, error) {
 		config = &Config{}
 	}
 
-	params := winrm.DefaultParameters()
+	connectTimeout := winrm.DefaultParameters.Timeout
+	if config.ConnectTimeout.Seconds() > 0 {
+		connectTimeout = iso8601.FormatDuration(config.ConnectTimeout)
```

Comment:
Thank you!  I knew I was going to confuse the timeouts somewhere.  This change is in the vendor directory, so I need to submit it up the packer-community repo for inclusion.  I will do that.